libkpod/image/copy.go

```diff
@@ -50,6 +60,10 @@ type CopyOptions struct {
 	// strip or add signatures to the image when pushing (uploading) the
 	// image to a registry.
 	common.SigningOptions
+	// Quiet suppresses the output when a push or pull happens
+	Quiet bool
+	// OciImage is the user specified image name that should be loaded
+	OciImage string
```

Comment:
`loadImage` now does the mapping, this member is no longer used.

Reply:
right, I missed that. Fixed.